ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionIndexing.scala

```diff
@@ -47,6 +40,10 @@ object TransactionIndexing {
           createArgumentsByContract += ((create.coid, createArgument))
           createArguments += ((nodeId, createArgument))
           createKeyValue.foreach(key => createKeyValues += ((nodeId, key)))
+          createKeyValue.foreach(_ => createKeyHashes += ((nodeId, create.key
+            .map(convertLfValueKey(create.templateId, _))
+            .map(_.hash.bytes.toByteArray)
```

Comment:
Why two consecutive maps instead of one?

Reply:
Purely Code style considerations: I cherish the threading style: easy and natural declaration of transformation steps with function literals. No need to introduce names for function parameters/function names/values. :)
If you disagree, or have a better approach please let me know!